.github/workflows/xtf-maven-release.yml

```diff
@@ -30,11 +30,11 @@ jobs:
         uses: whelk-io/maven-settings-xml-action@v12
         with:
           servers: '[{ "id": "jboss-releases-repository", "username": "${env.JBOSS_REPO_USER}", "password": "${env.JBOSS_REPO_PASSWORD}" },{ "id": "jboss-snapshots-repository", "username": "${env.JBOSS_REPO_USER}", "password": "${env.JBOSS_REPO_PASSWORD}" }]'
-      - name: Import GPG Key
-        uses: crazy-max/ghaction-import-gpg@v1
-        env:
-          GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
-          PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+#      - name: Import GPG Key
+#        uses: crazy-max/ghaction-import-gpg@v1
+#        env:
+#          GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
+#          PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
       - name: Maven Deploy
         run: mvn -X deploy -DskipTests -Prelease
```

Comment:
You will need to remove `-Prelease` for gh action to work. This will skip gpg-maven-plugin from triggering. Well, we should have 2 GH actions for this. One for creating tag when it will run this workflow and the second one just for branches and snapshots versions where gpg signing will be omitted.